assets/templates/index.html

```diff
@@ -14,7 +14,7 @@ <h2>Pirate Radio Bot <span class="version">v0.3.0</span></h2>
 		{{- range .}}
 		<div id="suggestion-{{.ID}}" class="suggestion" onclick="sessionStorage[this.id]=this.classList.toggle('bookmark');">
 			<p>{{.Message}}</p>
-			<span>Suggested by <em>{{.User}}</em> // <a href="{{.Link}}" target="_blank"><em>Play</em></a> // <a href="/play?id={{.ID}}"><em>Remove</em> and add to played</a> // <a href="/delete?id={{.ID}}"><em>Remove</em></a></span>
+			<span>Suggested by <em>{{.User}}</em> // <a href="{{.Link}}" target="_blank"><em>Play</em></a> // <a href="/play?id={{.ID}}"><em>Remove</em> and add to played</a> // <a href="/delete?id={{.ID}}"><em>Remove</em></a> // <a href="/purge?id={{.ID}}"><em>Remove to this point</em></span>
```

Comment:
I'd be more inclined to have it as "remove earlier suggestions", that way Kathleen can search for the first suggestion matching the description and click it rather than its predecessor. Also, see my comment on the other one w.r.t. saving the name "purge".